src/index.html

```diff
@@ -2,16 +2,203 @@
+        <div class="formField">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+            </label>
+        </div>
+
+        <div class="formField">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            autocomplete="off"
+            required
+          >
+            </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldSet">
+        <legend>An interesting fact about you!</legend>
+        <div class="formField">
+              Do you love cats?
+          <input
+            type="radio"
+            name="quiz"
+            value="Yes"
+            required
+            id="yes"
+          >
+          <label for="yes">Yes</label>
+          <input
+            type="radio"
+            name="quiz"
+            value="No"
+            required
+            id="no"
+          >
+          <label for="no">No</label>
+        </div>
+
+        <div class="formField">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            name="color"
+            id="color"
+            color="black"
+            required
+          >
+            </label>
+        </div>
+
+        <div class="formField">
+          <label for="time">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="time"
+            id="time"
+            step="2"
+            required
+          >
+        </div>
+
+        <div class="formField">
+          <label for="select">What are your favorite brand of cars?</label>
+          <select id="select" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+
+        <div class="formField">
+          <label for="range">How do you rate our work?</label>
+          <input type="range"
+          id="range"
+          value="0"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldSet">
+        <legend>Additional info</legend>
+        <div class="formField">
+          <label for="comments">Comments:</label>
+          <textarea
+          name="textarea"
+          id="comments"
+          autocomplete="off"
+         ></textarea>
+        </div>
+
+        <div class="formField">
+          <label for="reccomendation">Would you reccomend us?</label>
+          <select id="reccomendation">
+            <option value="yes" selected>yes</option>
+            <option value="no">no</option>
+          </select>
+        </div>
+      </fieldset>
+      <button type="submit" class="submitButton">Submit</button>
```

Comment:
You don't have such class, remove it
```suggestion
      <button type="submit">Submit</button>
```